guides/common/modules/proc_configuring-server-for-use-with-dhcpd.adoc

```diff
@@ -0,0 +1,93 @@
+[id="configuring-server-for-use-with-dhcpd"]
+= Configuring {ProjectServer} for use with dhcpd
```

Comment:
This is a very long procedure. I thought about how to make it look shorter and thought that you could perhaps split off the first step (install packages) into `.Prerequisites` and the last step (update configuration of that smartproxy) into `.Next steps`. Just a suggestion, though, it might not be a big enough improvement to justify the effort of doing this here and in all the other procedures -- your call :)

Reply:
I decided not to pursue this too much anymore and instead start the process of retiring the whole plugin. Fixing it up will likely cost too much time and mean we can't make the deadline. It was already poor quality before.

Reply:
Okay, let's pretend I didn't post this then :) Resolving.